sonar-application/src/main/assembly/conf/sonar.properties

```diff
@@ -217,9 +217,60 @@
 #--------------------------------------------------------------------------------------------------
 # LOGGING
 
-# Level of logs. Supported values are INFO(default), DEBUG and TRACE (DEBUG + SQL + ES requests)
+# SonarQube produces logs in 4 logs files located in the same directory (see property sonar.path.logs below),
+# one per JVM: App (main process), Web Server (aka. Web), Compute Engine (aka. CE) and Elastic Search (aka. ES)
```

Comment:
well... SQ has actually up to 5 processes : on windows the wrapper runs as a 5th process.
But SQ never has more than 4 JVMs.